elements/lisk-p2p/test/integration/duplicate_peers.ts

```diff
@@ -26,53 +26,9 @@ describe('Disconnect duplicate peers', () => {
 	let firstPeerDuplicate: OutboundPeer;
 	let firstP2PNode: P2P;
 	let existingPeer: InboundPeer;
-	const NETWORK_START_PORT = 5000;
-	const NETWORK_PEER_COUNT = 10;
-	const POPULATOR_INTERVAL = 50;
-	const DEFAULT_MAX_OUTBOUND_CONNECTIONS = 20;
-	const DEFAULT_MAX_INBOUND_CONNECTIONS = 100;
 
 	beforeEach(async () => {
-		p2pNodeList = [...new Array(NETWORK_PEER_COUNT).keys()].map(index => {
-			// Each node will have the previous node in the sequence as a seed peer except the first node.
-			const seedPeers =
-				index === 0
-					? []
-					: [
-							{
-								ipAddress: '127.0.0.1',
-								wsPort: NETWORK_START_PORT + index - 1,
-							},
-					  ];
-
-			const nodePort = NETWORK_START_PORT + index;
-			return new P2P({
-				connectTimeout: 100,
-				ackTimeout: 200,
-				rateCalculationInterval: 10000,
-				seedPeers,
-				wsEngine: 'ws',
-				populatorInterval: POPULATOR_INTERVAL,
-				maxOutboundConnections: DEFAULT_MAX_OUTBOUND_CONNECTIONS,
-				maxInboundConnections: DEFAULT_MAX_INBOUND_CONNECTIONS,
-				nodeInfo: {
-					wsPort: nodePort,
-					nethash:
-						'da3ed6a45429278bac2666961289ca17ad86595d33b31037615d4b8e8f158bba',
-					version: '1.0.1',
-					protocolVersion: '1.1',
-					minVersion: '1.0.0',
-					os: platform(),
-					height: 0,
-					broadhash:
-						'2768b267ae621a9ed3b3034e2e8a1bed40895c621bbb1bbd613d92b9d24e54b5',
-					nonce: `O2wTkjqplHII${nodePort}`,
-				},
-			});
-		});
-		await Promise.all(p2pNodeList.map(async p2p => await p2p.start()));
-
-		await wait(1000);
+		p2pNodeList = await createNetwork({});
```

Comment:
probably `createNetwork` could've have default to `{}` so that it doesn't need to provide, but not so important point